config/initializers/devise.rb

```diff
@@ -236,6 +236,7 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :linkedin, ENV['LINKEDIN_KEY'], ENV['LINKEDIN_SECRET']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.